commitizen/commands/changelog.py

```diff
@@ -115,18 +127,26 @@ def write_changelog(
                 changelog_out = changelog_hook(changelog_out, partial_changelog)
             changelog_file.write(changelog_out)
 
+    def export_template(self):
```

Comment:
nitpick

```suggestion
    def export_template(self) -> None:
```

Reply:
Other methods don't use this syntax. I think this one should be handled in another PR setting `ruff` and `mypy` accordingly.